### ISSUE TITLE
"zim-inserted-object" doesn't work for normal images.

### ISSUE THREAD

**Issue by k4nzdroid** [zim-desktop-wiki/zim-desktop-wiki#1421]
### What happened:
We could use "zim-inserted-object" to add a border to images, but this only works for images generated by plugins.

As the following example:

With gtk.css:
```
#zim-inserted-object {
    border: 2px solid #ccc;
}
```

 The first image generated by the Diagram plugin has a border. And the second one is inserted directly, which is no border.

![image](https://user-images.githubusercontent.com/40691275/111057095-9ea27900-84bf-11eb-9135-ec662f0d3585.png)

### Cause

The second image is inserted as a Pixbuf, and the first one is inserted as an ImageFileWidget. But the CSS property is only added to ImageFileWidget.

### What we expected to happen:

"zim-inserted-object" works for both.

Is there any way to solve this?

**Comment by jaap-karssenberg** [comment]
Yes, we plan to refactor images to become "objects" as well, but will take
one or two more releases before I get there

**Comment by jaap-karssenberg** [comment]
Consider this a duplicate of #324 (not exact duplicate, but do not need separate tracking)
